server/src/main/resources/application.example.yml

```diff
@@ -12,38 +12,79 @@
 
+#  the default port number of the seata server, default 7091, can be modified
 server:
   port: 7091
 
+
+#  the instance name of seata after it is registered with the registry cannot be duplicated
 spring:
   application:
     name: seata-server
 
+#  log configuration
 logging:
+  #  a logback configuration file that defines which components are used to collect logs
   config: classpath:logback-spring.xml
+  #  output the log to the path specified by file, where ${user.home}
+  #  represents a path that has been defined in the system, for example,
+  #  the user logged in to the Linux system is named seata_user,
+  #  then the path represented by user.home is /home/seata_user,
+  #  so this configuration means to output logs to /home/seata_user/logs/seata,
+  #  See logback-spring.xml for details
   file:
     path: ${log.home:${user.home}/logs/seata}
+  #  The components that process logs, these two are turned off by default,
+  #  file-appender and console-appender are enabled by default,
+  #  and the logback-spring .xml can be seen for details
   extend:
+    #  turn on logstash, the configuration file is under logback in the current directory,
+    #  the following is its working process
+    #  App(Logback LogstashTcpSocketAppender) -> Logstash -> Elasticsearch pipeline
     logstash-appender:
       destination: 127.0.0.1:4560
+    #  turn on kafka, the configuration file is under logback in the current directory,
+    #  the following is its working process
+    #  App(Logback KafkaAppender) -> Kafka -> Logstash -> Elasticsearch pipeline
     kafka-appender:
       bootstrap-servers: 127.0.0.1:9092
       topic: logback_to_logstash
 
+#  the account and password of the seata console
+console:
+  user:
+    username: seata
+    password: seata
+
+#  basic configuration information for seata
 seata:
+  #  seata's configuration center
   config:
-    # support: nacos 、 consul 、 apollo 、 zk  、 etcd3
+    # the type of configuration center,support: nacos 、 consul 、 apollo 、 zk  、 etcd3
     type: file
+    # type is the configuration when the file is made
+    file:
+      # write configuration information to the file.conf file
+      name: file.conf
+    # type is the configuration when the nacos is made
     nacos:
+      # nacos service address
       server-addr: 127.0.0.1:8848
+      # The namespace is generally a development environment, test environment, and production environment,
+      # and if it is not set, it is in the public space by default
       namespace:
+      # in nacos, services of the same type are generally grouped, and the role here is to register seata into SEATA_GROUP group
       group: SEATA_GROUP
-      username:
-      password:
-      context-path:
+      # the account and password of the nacos console
+      username: nacos
+      password: nacos
+      # for the path of nacos health check, you need to configure an open web endpoint
+      context-path: /${server.servlet.context-path}/actuator
```

Comment:
the default value is `${server.servlet.context-path}/actuator` ?  It's not a health check as I understand it.

Reply:
You can modify on this basis, and I will continue to review the modified code.